cmd/serve.go

```diff
@@ -58,6 +61,21 @@ func serve(_ context.Context) {
 		engineOpts = append(engineOpts, storage.WithTracing(config.Config.CRDB))
 	}
 
+	auditpath := viper.GetString("audit.log.path")
+	if auditpath == "" {
+		logger.Fatal("failed starting server. Audit log file path can't be empty")
```

Comment:
In tenant-api we print a warning and don't enable the middleware if no log path is provided. Should we do the same here or should it be a hard requirement to have this and therefore change it in tenant-api?
https://github.com/infratographer/tenant-api/blob/7b13895fd1ab8d9f0e2ab509dd1456aad2bb318a/cmd/server.go#L141-L157

Reply:
good point - i think we should make it optional for this